geps/gep-1709/index.md

```diff
@@ -370,84 +389,58 @@ implementation:
   organization: acme
   project: operator
   url: https://acme.com
-  version: v0.91.0
   contact:
   - @acme/maintainers
-date: "2022-09-28 20:29:41+00:00"
-gatewayAPIVersion: v0.6.2
-gatewayAPIChannel: standard
+  version: v0.9.0
+date: "2023-02-28 20:29:41+00:00"
+apiVersion: v0.8.0
+apiChannel: standard
+mode: plain
 profiles:
-  - name: tcp
+  profiles:
```

Comment:
Are there nested profiles here?

```
profiles:
  profiles:
    - name: tcp
    - name: http
```

Reply:
Nope, it's an error in the yaml. Good catch!

Reply:
fixed